### ISSUE TITLE
Release 6.0 - iOS deep link event won't fire listener

### ISSUE THREAD

**Issue by betorobson** [urbanairship/phonegap-ua-push#135]
On iOS looks like deep link event is not firing.

My device is running iOS 9.3.2
Xcode 8

I sent this push:
```json
{
"audience": "all",
"device_types": "all",
"notification": {
  "alert": "Message with deep link 017.",
  "actions": {"open": { "type": "deep_link", "content": "my deep link" }}
 }
}
```
When I tap on notification and app pops us, should I see a log trough the Safari Web Inspector isn't it?
Same behaviour is expected even if app is in foreground? 

Here is my listener:
```javascript
document.addEventListener(
	'urbanairship.deep_link',
	function(dl){console.log(dl)},
	false
);
```

**Comment by rlepinski** [comment]
Deep links are not fired by default on iOS in the foreground. Are you receiving the deep link event when you tap the notification?

**Comment by betorobson** [comment]
Actually I'm not receiving the deep link event when I tap the notification. This is the reason why asking help.

It works perfect on Android.

**Comment by rlepinski** [comment]
@betorobson ok, we will look into it.

**Comment by rlepinski** [comment]
@betorobson I was able to receive a deep link when tapping the notification. Do you have any logs you can post?

**Comment by betorobson** [comment]
Here is a part of my code from app.js
```javascript
	onDeviceReady: function() {
		console.log('device ready');

		document.addEventListener(
			'urbanairship.registration',
			function(event){
				console.log([
					'urbanairship onRegistration',
					event
				]);
			},
			false
		);

		document.addEventListener(
			'urbanairship.push',
			function(event){
				console.log([
					'urbanairship onPushReceived',
					event
				]);
			},
			false
		);

		document.addEventListener(
			'urbanairship.deep_link',
			function(event){
				console.log([
					'urbanairship onDeepLinkReceived',
					event
				]);
			},
			false
		);

	}
```

Device connected to safari web inspector and showing logs, I got this when I tap on push with deep link.
```
[Log] app resumed (app.js, line 67)
[Log] Firing document event: urbanairship.registration (UrbanAirship.js, line 121)
[Log] ["urbanairship onRegistration", Event] (2) (app.js, line 34)
[Log] Firing document event: urbanairship.inbox_updated (UrbanAirship.js, line 121)
```

**Comment by betorobson** [comment]
I also tried the getLaunchNotification method on redume app but doen't work. :-(

It looks likes plugin doesn't fires urbanairship.deep_link event on WebView and also doesn't fill getLaunchNotification arguments with the received push.

Pleaseeeeee any idea?

My onResume event listener
```javascript
	onResume: function() {
		console.log('app resumed');
		UAirship.getLaunchNotification(true, function(event){
			console.log('urbanairship getLaunchNotification onResume', event);
		});
	},
```

Log
`
[Log] urbanairship getLaunchNotification onResume – {message: "", extras: {}} (app.js, line 73)
`

**Comment by rlepinski** [comment]
I was unable to reproduce on iOS 10. I do not have an iOS 9 device handy at the moment so I won't be able to try again until the morning. Could you zip your xcode project and email it to me ryan.lepinski@urbanairship.com. Then I can debug your exact output.

**Comment by rlepinski** [comment]
I think i found the issue on the launch notification. I'll have it fixed first thing in the morning. Still no idea why the deep link event is not working for you. Sorry about the problematic update.

**Comment by betorobson** [comment]
No worries, I believe this will can help a lot of people in the future. I'm extremely thankfully of you helping me on this issue.

**Comment by rlepinski** [comment]
Ok, It looks like on iOS we expect a url but android it can be any string. Ill update to relax those requirements. If you send something like "whatever://cool" as the deep link value it will work fine.

**Comment by rlepinski** [comment]
Fixed in 6.0.1
